src/payment/strategies/googlepay/googlepay-payment-strategy.ts

```diff
@@ -123,6 +148,38 @@ export default class GooglePayPaymentStrategy implements PaymentStrategy {
         return Promise.reject(new OrderFinalizationNotRequiredError());
     }
 
+     async verifyCard(methodId: string , amount: number,  payment: any): Promise<GooglepayVerifyPayload>  {
+        if (methodId === 'googlepaybraintree') {
```

Comment:
I'm not a big fan of hardcoding like here, would we use `PaymentStrategyType.BRAINTREE_GOOGLE_PAY` for such case?

Reply:
We have it here too https://github.com/bigcommerce/checkout-sdk-js/pull/1240/files#diff-ffa8fb97c784b953d53f6ae3923e8869b16c410016854e84f3fd4dd0c6e153b7R194

Reply:
Replaced all payment methods strings with constants

Reply:
Do you mean `src/payment/strategies/googlepay/googlepay-payment-strategy.ts` ? IMHO, `payment-strategy-type.ts` is a real place you can trust, and any other changes in payment naming like this place could broke this piece of code potentially :) But if it OK we could skip this

Reply:
@bc-fetisov no, I agree with you. My comment was a reminder to not forget to add other googlepay stratagies to the constants.